lib/proxy/log_buffer/decorator.rb

```diff
@@ -10,17 +10,15 @@ def initialize(logger, buffer = Proxy::LogBuffer::Buffer.instance)
 
     def add(severity, message = nil, progname = nil, backtrace = nil, a_module = 'core', &block)
```

Comment:
But that's the way it's defined in Ruby Core, I want to keep the exact same interface in the decorator.